### DISCUSSION TITLE
Norns on a Raspberry Pi 3 Model A+

### DISCUSSION THREAD

**Discussion by bcostaaa01** [monome/norns#1632] [Q&A] [answered]
Hey @wrl @ngwese @simonvanderveldt @tehn @artfwo @scanner-darkly 👋

I have been planning to get my hands on a Raspberry Pi specifically to use with Norns, but I am in question whether to get an A+ or the B... 

Would you have any opinion about whether either of them would perform the job or to get a more robust model?

Bruno

**Comment by jonnybutler** [comment, marked as answer]
Raspberry Pi A isn’t supported: https://monome.org/docs/norns/shield/#requirements

**Reply by bcostaaa01** [reply]
Thank you for pointing me to the right documentation!
